styles/_mixin.scss

```diff
@@ -0,0 +1,98 @@
+
+
+@mixin Primary-btn {
```

Comment:
let's use 'button' word, not 'btn'. it doesn't save much space, but it's more difficult to read/write the code. I suggest 'button-primary'. etc